### ISSUE TITLE
Impossible to build tiles correctly with paris.osm on x86

### ISSUE THREAD

**Issue by xlqian** [valhalla/valhalla#2160]
Hi, 

I'm adding some tests on my bss PR and these tests failed only on x86... 

Here are some logs from valhalla_build_tiles:

```
...
2020/01/14 09:51:19.125623 [WARN] Max osmid exceeded bitset, resizing to fit id: 5526534229
2020/01/14 09:51:19.131962 [WARN] Max osmid exceeded bitset, resizing to fit id: 5526534246
2020/01/14 09:51:19.131973 [WARN] Max osmid exceeded bitset, resizing to fit id: 5526534240
2020/01/14 09:51:19.176768 [WARN] Max osmid exceeded bitset, resizing to fit id: 5528716184
2020/01/14 09:51:19.176975 [WARN] Max osmid exceeded bitset, resizing to fit id: 5526534239
2020/01/14 09:51:19.176988 [WARN] Max osmid exceeded bitset, resizing to fit id: 5526534231
2020/01/14 09:51:19.195280 [WARN] Max osmid exceeded bitset, resizing to fit id: 5526534234
2020/01/14 09:51:19.195432 [WARN] Max osmid exceeded bitset, resizing to fit id: 5528716212
2020/01/14 09:51:19.195653 [WARN] Max osmid exceeded bitset, resizing to fit id: 6879726889
2020/01/14 09:51:19.263052 [WARN] Max osmid exceeded bitset, resizing to fit id: 5528716518
2020/01/14 09:51:19.263221 [WARN] Max osmid exceeded bitset, resizing to fit id: 5749702237
2020/01/14 09:51:19.266731 [WARN] Max osmid exceeded bitset, resizing to fit id: 5749702260
2020/01/14 09:51:19.330437 [WARN] Max osmid exceeded bitset, resizing to fit id: 6227199875
2020/01/14 09:51:20.677705 [WARN] Max osmid exceeded bitset, resizing to fit id: 6383882928
... 
2020/01/14 09:51:22.946900 [ERROR] Cannot find node 5528716421 in way 4574420
2020/01/14 09:51:22.951842 [ERROR] Cannot find node 5295914129 in way 548171416
2020/01/14 09:51:22.951849 [ERROR] Cannot find node 5295914128 in way 548171416
2020/01/14 09:51:22.951856 [ERROR] Cannot find node 5295914127 in way 548171416
2020/01/14 09:51:22.951920 [ERROR] Do not add edge!
2020/01/14 09:51:22.951929 [ERROR] Cannot find node 5528716211 in way 576429481
2020/01/14 09:51:22.951934 [ERROR] Cannot find node 5528716213 in way 576429481
2020/01/14 09:51:22.951937 [ERROR] Do not add edge!
2020/01/14 09:51:22.952111 [ERROR] Do not add edge!
2020/01/14 09:51:22.952117 [ERROR] Cannot find node 6631842919 in way 705792797
2020/01/14 09:51:22.952148 [ERROR] Do not add edge!
2020/01/14 09:51:22.952154 [ERROR] Cannot find node 5526534443 in way 738914252
2020/01/14 09:51:22.952194 [ERROR] Do not add edge!
...
```

Here is the fix... https://github.com/valhalla/valhalla/pull/2161

**Comment by kevinkreiser** [comment]
we ran into this a couple of times but didnt realize it was coming from this spot in the code.. thank you for tracking that down!

**Comment by kevinkreiser** [comment]
fixed in #2161
